en/docs/integrations/integration-concepts.md

```diff
@@ -0,0 +1,98 @@
+# Concepts
+
+Before you start building integrations with Choreo, there are basic concepts that you need to understand. This section walks you through all the integration concepts you need to know. 
+ 
+## Integration
+
+Integration in Choreo is the process of connecting APIs, events, and streams to function as a single unit that can share information.
+
+
+## Integration Template
+
+An integration template in Choreo is a prebuilt sample designed to address a common integration requirement. Choreo provides a comprehensive set of integration templates for you to select and use depending on your requirement.
+
+
+## Trigger
+
+A trigger in Choreo is an event or an action that can cause a Choreo application to start executing.
+
+Choreo supports the following trigger types:
+
+ - **API**: Allows you to expose an application as an API endpoint. 
+ - **Manual**: Allows you to trigger an application manually.
+ - **Schedule**: Allows to trigger an application based on a schedule that you set. You can specify the schedule in the form of a cron expression.
+   <html><div class="admonition note">
```

Comment:
Shall we use "> **Note**: " instead of this HTML element?

Reply:
Btw do we need this section? We don't support cron based editor now.

Reply:
Removed the note for now. Need to validate how exactly we are going to have the cron expression in the UI and then add the note if necessary.